### PR TITLE
NN-5031 removed all Camel references and also added K8E Cronjobs

### PR DESCRIPTION
**Pull request by nagendramoj**

### CODE REVIEW

**Review by neilmendum — COMMENTED** [review]
Looks good - few comments

File: build.gradle.kts [hunk 1]
```diff
@@ -27,10 +26,6 @@ dependencies {
   implementation("org.springframework:spring-webflux")
   implementation("org.springframework.boot:spring-boot-starter-reactor-netty")
   implementation("org.springframework.retry:spring-retry")
-  implementation("org.apache.camel.springboot:camel-spring-boot:$camelVersion") // DO NOT UPDATE - BREAKS CAMEL
```
[the remark this answers is not in the document]
Comment (nagendramoj) [reply build.gradle.kts:30]: Done

File: src/main/resources/application-batch.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-quartz.enabled=true
+quartz.enabled=false
```
[the remark this answers is not in the document]
Comment (nagendramoj) [reply src/main/resources/application-batch.properties:1]: Done

File: helm_deploy/keyworker-api/templates/enable-new-nomis-cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    spec:
+      template:
+        spec:
+          containers:
+            - name: keyworker-api
+              image: ghcr.io/ministryofjustice/hmpps-devops-tools
+              args:
+                - /bin/sh
+                - -c
+                - curl --retry 2 -XPOST http://keyworker-api/add-users-to-new-nomis
```
[the remark this answers is not in the document]
Comment (nagendramoj) [reply helm_deploy/keyworker-api/templates/enable-new-nomis-cronjob.yaml:23]: Done

File: src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/BatchController.java [hunk 4]
```diff
@@ -8,37 +8,40 @@
...
+    private final KeyworkerBatchService keyworkerBatchService;
 
-    public BatchController(final ProducerTemplate producerTemplate) {
-        this.producerTemplate = producerTemplate;
-    }
+    private final KeyworkerStatsBatchService keyworkerStatsBatchService;
 
+    private final NomisBatchService nomisBatchService;
+
+    private final ReconciliationBatchService reconciliationBatchService;
```
[the remark this answers is not in the document]
Comment (nagendramoj) [reply src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/BatchController.java:38]: Done

File: helm_deploy/keyworker-api/templates/housekeeping-cronjob.yaml [hunk 5]
```diff
@@ -15,10 +15,10 @@ spec:
...
         spec:
           containers:
-            - name: keyworker-api
+            - name: housekeeping-keyworker-api
               image: ghcr.io/ministryofjustice/hmpps-devops-tools
               args:
                 - /bin/sh
                 - -c
-                - curl --retry 2 -XPUT http://keyworker-api/queue-admin/retry-all-dlqs
+                - curl --retry 2 -XPUT http://keyworker-api/batch/queue-admin/retry-all-dlqs
```
Comment (neilmendum) [inline helm_deploy/keyworker-api/templates/housekeeping-cronjob.yaml:23]: I don't think you need to update this URL - it isn't managed by `BatchController`

Comment (nagendramoj) [reply helm_deploy/keyworker-api/templates/housekeeping-cronjob.yaml]: Done

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**